tests/unittests/QuantizationTest.cpp

```diff
@@ -1962,7 +1956,7 @@ static NodeClass *findNodeKindOrReturnNull(Function *F) {
 template <class BackendClass>
 static void testProfileQuantizationOfFC(bool expectLoweredFC,
                                         bool rowwiseQuantizeFC) {
-  ExecutionEngine profileEE(BackendKind::Interpreter);
+  ExecutionEngine profileEE("Interpreter");
```

Comment:
Why do you leave "Interpreter" here, but removed it in many other places?

Reply:
Yeah, I'll remove in all. No particular reason.